rcl/test/rcl/test_lexer_lookahead.cpp

```diff
@@ -195,6 +195,42 @@ TEST_F(CLASSNAME(TestLexerLookaheadFixture, RMW_IMPLEMENTATION), test_accept)
   EXPECT_EQ(RCL_LEXEME_EOF, lexeme);
 }
 
+TEST_F(CLASSNAME(TestLexerLookaheadFixture, RMW_IMPLEMENTATION), test_accept_bad_arg)
+{
+  rcl_ret_t ret;
+  rcl_lexer_lookahead2_t buffer;
+  rcl_lexer_lookahead2_t buffer_not_ini = rcl_get_zero_initialized_lexer_lookahead2();
+  SCOPE_LOOKAHEAD2(buffer, "foobar/");
+
+  rcl_lexeme_t lexeme = RCL_LEXEME_NONE;
+  const char * lexeme_text;
+  size_t lexeme_text_length;
+
+  ret = rcl_lexer_lookahead2_accept(&buffer, &lexeme_text, &lexeme_text_length);
```

Comment:
@Blast545 nit: it'd be super helpful to have comments explaining what error condition is each assertion covering. It's not immediately obvious.

Reply:
Done ac4d561